website/src/blogs/2018-10-06-contributing-to-mayactl-cli-in-hacktoberfest-2018.md

```diff
@@ -0,0 +1,180 @@
+
+    NAME STATUS ROLES AGE VERSIONminikube Ready master 26m v1.10.0
+
+## Setting up OpenEBS on Minikube
+
+Once the kubernetes cluster is ready, it’s time to install openebs. Installing openebs on kubernetes is very easy; just apply the openebs-operator to the cluster using the following command:
+
+    kubectl appy -fhttps://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-operator.yaml.
+
+Applying the openebs-operator creates openebs resources in kubernetes. We can obtain the openebs-related resources in the openebs namespace, and we can verify it by typing the following command:
+
+    ashishranjan738@Ashish-PC:~$ kubectl get pods -n openebs
+    
+    NAME READY STATUS RESTARTS AGE
+    
+    cstor-sparse-pool-ni3o-8485bf44cd-fndj6 2/2 Running 0 5m
+    
+    maya-apiserver-674c77bb4c-dhnw5 1/1 Running 0 6m
+    
+    openebs-ndm-rd7hf 1/1 Running 0 6m
+    
+    openebs-provisioner-6d6c9ccf75-s57j8 1/1 Running 2 6m
+    
+    openebs-snapshot-operator-5cdd4ddc46-c2qj5 2/2 Running 0 6m
+
+## Obtaining Access to mayactl
+
+Once all of the pods are in the Running state, and once the maya-apiserver pod is in the Running state, we can access the mayactl binary by accessing the maya-apiserver pod’s shell. We can gain access to mayactl by using the following command:
+
+    kubectl exec -it <maya-api server pod name> -n openebs
+
+(e.g the pod name in my case is maya-apiserver-674c77bb4c-dhnw5). This will open the pod’s terminal over your system terminal. Typing mayactl version should produce something similar to this:
+
+    bash-4.3# mayactl version
+    
+    Version: 0.7.0-unreleased
+    
+    Git commit: 0fa41c79dde0f6378056093d62494cd2fbb1eea4
+    
+    GO Version: go1.11
+    
+    GO ARCH: amd64
+    
+    GO OS: linux
+    
+    m-apiserver url:  http://172.17.0.5:5656
+    
+    m-apiserver status: running
+    
+    Provider: KUBERNETES
+    
+    You can explore the current features of mayactl using the mayactl -help.command. Now we will look at where you can find the mayactl source code for hacking and how you can test the mayactl binary.
+
+## Exploring mayactl Code
+
+The source code of mayactl can be found in cmd/mayactl in the maya [repository](https://github.com/openebs/maya/tree/master/cmd/mayactl). mayactl uses [cobra](https://github.com/spf13/cobra) cli framework for its implementation, so knowledge of this will be very helpful when starting with [good-first issues](https://github.com/openebs/openebs/labels/good%20first%20issue).
+
+## Building the mayactl Binary
+
+After playing with the code, it’s time to compile the code to binary and test it against the maya-apiserver. For building the mayactl binary, you can use the make mayactl command. This will trigger the build process, and the newly-built binary can found in bin/maya under the maya directory that you have cloned. This will generate an output of ls -ltr bin/maya/ -ltr :
```

Comment:
```suggestion
After playing with the code, it’s time to compile the code to binary and test it against the `maya-apiserver`. For building the `mayactl` binary, you can use the make `mayactl` command. This will trigger the build process, and the newly-built binary can found in `bin/maya` under the `maya` directory that you have cloned. This will generate an output of `ls -ltr bin/maya/ -ltr` :
```